model/communication/Proxy.ttl

```diff
@@ -18,6 +18,10 @@ ids:Proxy a owl:Class;
         idsm:forProperty ids:proxyURI;
         idsm:constraint idsm:NotNull;
                   ];
+    idsm:validation [
+        idsm:forProperty ids:proxyAuthentication;
```

Comment:
Please verify, if we need to enforce auth for any proxy. This might be optional.

Reply:
For userAuthentication below, we allow `idsm:OneToMany`. Could this be an option here?

Reply:
You're right. A proxy could be used without authentication!